File: Pattern30.c
// E D C B A
// D C B A
// C B A
// B A
// A

#include<stdio.h>

void main(){
    int n;
    printf("Enter the number:");
    scanf("%d",&n);
    int val = 64+n;
    for(int i=1;i<=n;i++){
        int printable = val;
        for(int j=i;j<=n;j++){
            printf("%c ",(char)(printable--));
        }
        val--;
        printf("\n");
    }
}

File: Pattern30.go
// E D C B A
// D C B A
// C B A
// B A
// A

package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Printf("Enter the number:")
	fmt.Scanf("%d", &n)
	var val int = 64+n;
    for i:=1;i<=n;i++{
        printable := val;
        for j:=i;j<=n;j++{
			fmt.Printf("%c ",printable);
			printable = printable - 1
        }
        val--;
        fmt.Printf("\n");
    }
}